src/integration-test/java/uk/gov/ida/verifylocalmatchingserviceexample/rules/H2JDBIRule.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.ida.verifylocalmatchingserviceexample.rules;
+
+import io.dropwizard.testing.junit.DropwizardAppRule;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.core.Jdbi;
+import org.junit.rules.ExternalResource;
+import uk.gov.ida.verifylocalmatchingserviceexample.configuration.VerifyLocalMatchingServiceExampleConfiguration;
+
+public class H2JDBIRule extends ExternalResource {
```

Comment:
Can we have a more explanatory name for this? (What does H2 even mean here?)

Reply:
H2 is the in-memory database that we use for integration test. I will change it to TestDatabaseRule. Does that sound meaningful ?